src/BloomExe/MiscUI/ProblemReporterDialog.cs

```diff
@@ -332,6 +332,28 @@ private bool SubmitToYouTrack()
 							zip.Save();
 							AddAttachment(bookZip.Path);
 						}
+						catch (InvalidRequestException e)
+						{
+							// We get this rather unspecific exception (with the even more unhelpful message 'NoContent') if the attachment is too large.
+							// There might of course be other reasons.
+							bookZip.Detach();
```

Comment:
I don't know. It was present in the generic case below that handles all exceptions, so I presumed there was a good reason for it.
It's very possible that they were intended for debugging and both should be removed.